src/includes/getting-started-install/dart.mdx

```diff
@@ -1,4 +1,4 @@
 ```yml {filename:pubspec.yaml}
 dependencies:
-  sentry: ^4.0.0
+  sentry: ^4.0.0-beta.1
```

Comment:
Once we use `craft` and this is in the registry we can use the placeholder instead of hardcoding the version number.

Reply:
nice, yeah Craft is on my TODO